blockdag/dag.go

```diff
@@ -1366,6 +1366,23 @@ func (dag *BlockDAG) BlockConfirmationsByHashNoLock(hash *daghash.Hash) (uint64,
 	return dag.blockConfirmations(node)
 }
 
+// TxConfirmations returns the confirmations for the given outpoint, if it exists
+// in the DAG.
```

Comment:
"if it exists in the DAG" - not exactly. It might be already spend in the DAG.
I'd say something like "if it exists in the DAG's UTXO-Set" or something.